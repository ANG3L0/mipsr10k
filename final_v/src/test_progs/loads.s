/*
	

*/
	data = 0x1000
	lda	$r5,0
	lda	$r1,data
	#loop:	mulq	$r5,0x0a,$r2
	stq	$r2,0($r1)
	ldq	$r3,0($r1)
	#stq     $r3,0x100($r1)
	#addq    $r1,0x8,$r1
	#addq	$r5,0x1,$r5
	#cmple   $r5,0xf,$r4
	#bne     $r4,loop
	call_pal        0x555

