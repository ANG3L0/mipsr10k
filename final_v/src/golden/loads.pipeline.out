                                                                            D-MEM Bus &
Cycle:      IF      |     ID      |     EX      |     MEM     |     WB      Reg Result
    0:    4:-       |   0:-       |   0:-       |   0:-       |   0:-       
    1:    4:-       |   4:-       |   0:-       |   0:-       |   0:-       BUS_LOAD  MEM[0] accepted 1
    2:    4:-       |   4:-       |   4:-       |   0:-       |   0:-       
    3:    4:-       |   4:-       |   4:-       |   4:-       |   0:-       
    4:    4:-       |   4:-       |   4:-       |   4:-       |   4:-       
    5:    4:-       |   4:-       |   4:-       |   4:-       |   4:-       
    6:    4:lda     |   4:-       |   4:-       |   4:-       |   4:-       
    7:    8:-       |   4:lda     |   4:-       |   4:-       |   4:-       
    8:    8:-       |   8:-       |   4:lda     |   4:-       |   4:-       
    9:    8:-       |   8:-       |   8:-       |   4:lda     |   4:-       
   10:    8:-       |   8:-       |   8:-       |   8:-       |   4:lda     r5=0  
   11:    8:lda     |   8:-       |   8:-       |   8:-       |   8:-       
   12:   12:-       |   8:lda     |   8:-       |   8:-       |   8:-       
   13:   12:-       |  12:-       |   8:lda     |   8:-       |   8:-       BUS_LOAD  MEM[8] accepted 1
   14:   12:-       |  12:-       |  12:-       |   8:lda     |   8:-       
   15:   12:-       |  12:-       |  12:-       |  12:-       |   8:lda     r1=4096  
   16:   12:-       |  12:-       |  12:-       |  12:-       |  12:-       
   17:   12:-       |  12:-       |  12:-       |  12:-       |  12:-       
   18:   12:stq     |  12:-       |  12:-       |  12:-       |  12:-       
   19:   16:-       |  12:stq     |  12:-       |  12:-       |  12:-       
   20:   16:-       |  16:-       |  12:stq     |  12:-       |  12:-       
   21:   16:-       |  16:-       |  16:-       |  12:stq     |  12:-       BUS_STORE MEM[4096] = -1 accepted 1
   22:   16:-       |  16:-       |  16:-       |  16:-       |  12:stq     
   23:   16:ldq     |  16:-       |  16:-       |  16:-       |  16:-       
   24:   20:-       |  16:ldq     |  16:-       |  16:-       |  16:-       
   25:   20:-       |  20:-       |  16:ldq     |  16:-       |  16:-       BUS_LOAD  MEM[16] accepted 2
   26:   20:-       |  20:-       |  20:-       |  16:ldq     |  16:-       BUS_LOAD  MEM[4096] accepted 1
   27:   20:-       |  20:-       |  20:-       |  16:ldq     |  16:-       
   28:   20:-       |  20:-       |  20:-       |  16:ldq     |  16:-       
   29:   20:-       |  20:-       |  20:-       |  16:ldq     |  16:-       
   30:   20:-       |  20:-       |  20:-       |  16:ldq     |  16:-       
   31:   20:-       |  20:-       |  20:-       |  20:-       |  16:ldq     r3=-1  
   32:   20:halt    |  20:-       |  20:-       |  20:-       |  20:-       
   33:   24:-       |  20:halt    |  20:-       |  20:-       |  20:-       
   34:   24:-       |  24:-       |  20:halt    |  20:-       |  20:-       
   35:   24:-       |  24:-       |  24:-       |  20:halt    |  20:-       
   36:   24:-       |  24:-       |  24:-       |  24:-       |  20:halt    
